modules/shapefile/src/lib/parsers/parse-dbf.ts

```diff
@@ -290,7 +290,7 @@ function parseRow(
   fields: DBFField[],
   textDecoder: TextDecoder
 ): {[key: string]: any} {
-  const out = {};
+  const out = {} as {[key: string]: string | number | boolean | null};
```

Comment:
Does this work? E.g. typing the declaration instead of casting the initialization value?
```suggestion
  const out: {[key: string]: string | number | boolean | null} = {};
```

Reply:
It does, I update the code accordingly.